src/drive/web/modules/viewer/Panel/useBlocks.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import { hasCertifications } from 'drive/web/modules/viewer/helpers'
+import Certifications from './Certifications'
+
+// TODO add connector block
+const blocksSpecs = file => ({
+  certifications: {
+    condition: hasCertifications(file),
+    component: <Certifications file={file} />
+  }
+})
```

Comment:
Can you rename the function so that there is a verb ?

Nit: It seems this could actually be refactored a static object :

```
const blocksSpecs = {
  certifications: {
    condition: hasCertifications,
    component: Certifications
  }
}
```

I think it would make more sense, this way in "useBlocks", the condition would be called.